src/Microsoft.SqlTools.ServiceLayer/DacFx/Contracts/DeploymentOptions.cs

```diff
@@ -197,13 +205,13 @@ public class DeploymentOptions
 
         public DeploymentOptionProperty<object> IgnoreTablePartitionOptions { get; set; } // DW Specific
 
-        public DeploymentOptionProperty<object> AdditionalDeploymentContributorPaths { get; set; } = new DeploymentOptionProperty<object>();
+        public DeploymentOptionProperty<object> AdditionalDeploymentContributorPaths { get; set; } = new DeploymentOptionProperty<object>(null);
```

Comment:
I don't know what an appropriate values for these would be. Do you even want to initialize them here vs just leaving the property null to begin with?